specification/electronic-prescription-service-api.yaml

```diff
@@ -139,35 +139,31 @@ info:
     **In step 11:** Submit your completed SCAL to api.management@nhs.net.
     **In step 14:** To request production access, contact us at api.management@nhs.net.
     
+    ## Additional guidance
+    For further details on common HTTP status codes, see [5xx status codes](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide)
+    
+    ## Resources
+    Use the [Digital Medicines Implementation Guide](https://simplifier.net/guide/DigitalMedicines) to assist with your integration.
+    This is the FHIR specification for Digital Medicines starting with the assets required for an electronic prescription sent to the EPS.
+    
     ## Responses
     ### 2xx responses
-    **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
+    Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
```

Comment:
Should this still have ** at the end?

Reply:
No. The two ** at the end can be removed.